packages/nimble-components/CONTRIBUTING.md

```diff
@@ -77,15 +77,33 @@ Before building a new component, 3 specification documents need to be created:
 
 ## Develop new components
 
-### Marking a component as in development
+### Marking a component as incubating
```

Comment:
Do we need a section in nimble-blazor and nimble-angular as well?

I think we should make new incubating angular components go under a secondary entry point ie `@ni/nimble-angular/incubating/table-column/select`;

Reply:
Closing since we decided not to reflect incubating status in the path/entrypoint/namespace.